safe-global/web-core
# refactor: use `UrlObject`s for internal links

## What it solves

Difficult to understand link structure

## How this PR fixes it

All internal links have been reformatted to `UrlObject`s.

## How to test it

The following links should work as before:

- Safe token widget
- Dashboard: assets, NFTs, pending and view all transactions
- Safe Apps: opening internal/external apps, sharing and demoing them
- Opening Safes via the sidebar
- Notifications: updating the Safe, transaction lifecycles and invalid mastercopy (which should open in a new tab).
